code/game/verbs/who.dm

```diff
@@ -1,63 +1,94 @@
-
 /client/verb/who()
 	set name = "Who"
 	set category = "OOC"
 
-	var/msg = "<b>Current Players:</b>\n"
+	var/msg = "<meta charset='UTF-8'><h2><b>Curret Players:</b></h2>\n"
 
 	var/list/Lines = list()
 
+	var/living = 0
+	var/dead = 0
+	var/observers = 0
+	var/lobby = 0
+	var/living_antags = 0
+	var/dead_antags = 0
+
 	if(check_rights(R_INVESTIGATE, 0))
+		log_admin("[key_name(usr)] checked advanced who in-round.")
```

Comment:
Unlogged change.

Reply:
Also seems kinda unnecessary, why does the admin team need to know when you use who?

Reply:
It looks like it's specifically for admin-who, which does show stuff like antag status.